ups/eupspkg.cfg.sh

```diff
@@ -44,12 +44,15 @@ install()
         # Install packages on which the stack is known to depend
 
         export PATH="$PREFIX/bin:$PATH"
-        local tmpdir=${TMPDIR:-/tmp}
         local reqfile='conda_packages.txt'
         local baseurl='https://raw.githubusercontent.com/lsst/lsstsw/master/etc/'
-        (cd "$tmpdir"; $CURL -# -L -O "${baseurl}/${reqfile}")
+        local tmpfile
+        tmpfile=$(mktemp -t "${reqfile}.XXXXXXXX")
+        # shellcheck disable=SC2064
+        trap "{ rm -rf $tmpfile; }" EXIT
```

Comment:
and on OS X there is the `-u` option which removes the temp file. I don't know if that is available on Linux.